src/AspNetCoreModuleV2/CommonLib/aspnetcore_msg.mc

```diff
@@ -177,19 +177,19 @@ Language=English
 .
 
 Messageid=1025
-SymbolicName=ASPNETCORE_EVENT_GENERAL_INFO_MSG
+SymbolicName=ASPNETCORE_EVENT_GENERAL_INFO
```

Comment:
Why this change?

Reply:
I can make is a separate PR. Events shouldn't have MSG at the end (see others in file).